### PR TITLE
Add examples to README.md

### PR DESCRIPTION
**Pull request by thekostya**
Add examples:
- How to use `in` clause
- How to use nested types
- How to use multiple inserts and updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +185,49 @@ func (r *DeleteAuthorsReq) Query() string {
...
+func (r *DeleteAuthorsReq) Query() string {
+	return `DELETE FROM authors WHERE tags=ANY(@tags::UUID[])`
+}
+```
+
+##  Nested types
+
+```go
+type Author struct {
+    ID   int64  `dql:"id"`
```
Comment (izaurio) [inline README.md]: `dql`?

Comment (thekostya) [reply README.md]: typo

File: README.md [hunk 2]
```diff
@@ -159,6 +185,49 @@ func (r *DeleteAuthorsReq) Query() string {
...
+func (r *CreateBookReq) ProcessRow(rowMap sal.RowMap) {
+	rowMap.Set("author_id", r.Book.Author.ID)
+	rowMap.Set("book_id",   r.Book.ID)
+	rowMap.Set("book_name", r.Book.Name)
+	rowMap.Set("book_desc", r.Book.Desc)
+}
+
+func (r *CreateBookReq) Query() string {
+	return `INSERT INTO books (id, author_id, name, description)
+	VALUES (@book_id, @author_id, @book_name, @book_desc)`
```
Comment (izaurio) [inline README.md]: I think it produces missunderstanding: `Desc`, `description`, `@book_desc`

Comment (thekostya) [reply README.md]: check it

**Review by izaurio — APPROVED** [review]

**Pull request merged**